docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,274 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
+########################################
+
+With the Courses API **Courses** resource, you can complete the
+following tasks.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Get a list of courses:
+
+**********************
+Get a List of Courses
+**********************
+
+The endpoint to get a list of courses is ``/api/courses/v1/courses/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of the courses that are visible to a specific user. If the request
+is made by an anonymous user, a username is not required.
+
+=====================
+Example Requests
+=====================
+
+GET /api/courses/v1/courses/?&username=<username>
+
+GET /api/courses/v1/courses/
+
+=====================
+Query Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
```

Comment:
Change "made by an anonymous user" -> "made for an anonymous user".

The implementation of the API changed since it was originally implemented.  Initially, we required that the requesting user had to be anonymous if the username wasn't supplied.  Later, we changed it so that we automatically assume the requested user is anonymous when a username isn't supplied, even if the requesting user isn't anonymous.

Also, please note that another user's username can be supplied here, as long as the requesting user has staff access.